output/clickhouse.go

```diff
@@ -64,14 +65,16 @@ func (c *ClickHouse) Init() error {
 }
 
 // Send a batch to clickhouse
-func (c *ClickHouse) Send(batch model.Batch, callback func(batch model.Batch) error) {
+func (c *ClickHouse) Send(batch *model.Batch, callback func(batch *model.Batch) error) {
 	// TODO workerpool parallel
 	statistics.FlushBatchBacklog.WithLabelValues(c.taskCfg.Name).Inc()
-	c.loopWrite(batch, callback)
+	_ = util.GlobalWorkerPool2.Submit(func() {
```

Comment:
Let's rename it to `GlobalSinkWorkerPool`

Reply:
Renamed to `GlobalWritingPool`.